IGI.Lab3/CarPark.WebUI/Areas/Identity/IdentityInitializer.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Threading.Tasks;
+using CarPark.DAL.Models;
+using Microsoft.AspNetCore.Identity;
+
+namespace CarPark.WebUI.Areas.Identity
+{
+    public class IdentityInitializer
+    {
+        private const string _adminEmail = "admin@mail.com";
+        private const string _adminPass = "Qwerty123";
+
+        public static async Task InitializerAsync(UserManager<ApplicationUser> userManager, RoleManager<IdentityRole> roleManager)
+        {
+            if (await roleManager.FindByNameAsync(Roles.Admin) == null)
+            {
+                await roleManager.CreateAsync(new IdentityRole(Roles.Admin));
+            }
+
+            if (await roleManager.FindByNameAsync(Roles.User) == null)
+            {
+                await roleManager.CreateAsync(new IdentityRole(Roles.User));
+            }
+
+            if (await userManager.FindByNameAsync(_adminEmail) == null)
+            {
+                var admin = new ApplicationUser() { Email = _adminEmail, UserName = _adminEmail};
```

Comment:
Redundant ()

Reply:
I do not understand what you mean here

Reply:
What you cannot understand?
This '()' useless here.

Reply:
Aaaa
fix